src/Symfony/Component/Dotenv/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+5.4.0
```

Comment:
5.4

Reply:
should i fix the other version numbers in der file as well?

Reply:
No, just this one. We've changed our format with 5.3, but we don't want to change old changelogs.